### DISCUSSION TITLE
The full-screen preview can be minimized by clicking on it again, instead of clicking the X to close it.

### DISCUSSION THREAD

**Discussion by 779575267** [anapnoe/stable-diffusion-webui-ux#124] [Ideas]
The full-screen preview can be minimized by clicking on it again, instead of clicking the X to close it. just like A1111，It's tiring to move the mouse to the top right corner to find the X every time, especially on a large screen. Thank you so much.

**Comment by anapnoe** [comment]
clicking is reserved for panning and swipe to next previous if you batch render 
I will add some shortcuts for this kind of situations that will help do things faster

**Reply by halr9000** [reply]
> clicking is reserved for panning and swipe to next previous if you batch render I will add some shortcuts for this kind of situations that will help do things faster

What do you think about a key handler for ESC?

**Comment by drdancm** [comment]
How do you get a Full Screen Preview ????
